django/src/rdwatch/views/model_run.py

```diff
@@ -163,67 +151,75 @@ def get_queryset():
         )
+        .values()
         .annotate(
-            json=JSONObject(
-                id='pk',
-                title='title',
-                created='created',
-                expiration_time='expiration_time',
-                performer=Subquery(  # prevents including "performer" in slow GROUP BY
-                    lookups.Performer.objects.filter(
-                        pk=OuterRef('performer_id')
-                    ).values(
-                        json=JSONObject(
-                            id='id',
-                            team_name='description',
-                            short_code='slug',
-                        )
-                    ),
-                    output_field=JSONField(),
-                ),
-                region=Subquery(  # prevents including "region" in slow GROUP BY
-                    Region.objects.filter(pk=OuterRef('region_id')).values('name')[:1],
-                ),
-                downloading=Coalesce(
-                    Subquery(
-                        SatelliteFetching.objects.filter(
-                            siteeval__configuration_id=OuterRef(
-                                'evaluation_configuration'
-                            ),
-                            status=SatelliteFetching.Status.RUNNING,
-                        )
-                        .annotate(count=Func(F('id'), function='Count'))
-                        .values('count')
-                    ),
-                    0,  # Default value when evaluations are None
+            performer=Subquery(  # prevents including "performer" in slow GROUP BY
+                lookups.Performer.objects.filter(pk=OuterRef('performer_id')).values(
+                    json=JSONObject(
+                        id='id',
+                        team_name='description',
+                        short_code='slug',
+                    )
                 ),
-                parameters='parameters',
-                numsites=Count('evaluations__pk', distinct=True),
-                score=Avg('evaluations__score'),
-                evaluation='evaluation',
-                evaluation_run='evaluation_run',
-                timestamp=ExtractEpoch(Max('evaluations__timestamp')),
-                timerange=JSONObject(
-                    min=ExtractEpoch(Min('evaluations__start_date')),
-                    max=ExtractEpoch(Max('evaluations__end_date')),
-                ),
-                bbox=BoundingBoxGeoJSON('evaluations__geom'),
-                proposal='proposal_val',
-                adjudicated=Case(
-                    When(
-                        ~Q(proposal_val=None),  # When proposal has a value
-                        then=JSONObject(
-                            proposed=Coalesce(Subquery(proposed_count_subquery), 0),
-                            other=Coalesce(Subquery(other_count_subquery), 0),
-                        ),
-                    ),
-                    default=None,
```

Comment:
Any way we can keep this.  It's a simple thing that should only be run when the proposal_val is non-null.  It is used for the annotators to see the current status of a model run and indicate the proposed and remaining sites?

They don't need to be in the JSONB object.

Reply:
My bad, somehow I missed that. The intent of this PR is to keep all the original fields, so I added that back in.